includes/class-wc-install.php

```diff
@@ -560,7 +560,7 @@ private static function get_schema() {
 ) $collate;
 CREATE TABLE {$wpdb->prefix}woocommerce_downloadable_product_permissions (
   permission_id BIGINT UNSIGNED NOT NULL auto_increment,
-  download_id varchar(32) NOT NULL,
+  download_id varchar(36) NOT NULL,
```

Comment:
Do we need an update script for this, too? Otherwise old DBs won't properly handle the new download IDs?

Reply:
dbdelta handles it